jupyter/Dockerfile

```diff
@@ -1,21 +1,23 @@
-# Image: pubnative/jupyter:0.7
+# Image: pubnative/pyspark-ci
+# Cf: Makefile.
 
-FROM jupyter/all-spark-notebook:f646d2b2a3af
+FROM jupyter/all-spark-notebook:7d427e7a4dde
```

Comment:
@ValentinMouret may be better to have label instead of commit id?

Reply:
I disagree. A commit ID makes it easier to get the Dockerfile this image is based on.